src/app/shared/subscription-feed/subscription-feed.component.css

```diff
@@ -0,0 +1,78 @@
+.subscription-feed-icon-container {
+  flex-shrink: 0;
+  border-radius: 1rem;
+  margin-right: 1rem;
+}
```

Comment:
whats the thing with "container". either use a syntax like BEM or leave out unneccesary information from css classes (makes css very messy)